lib/pure/strutils.nim

```diff
@@ -84,31 +84,39 @@ from std/private/strimpl import cmpIgnoreStyleImpl, cmpIgnoreCaseImpl, startsWit
 
 
 const
-  Whitespace* = {' ', '\t', '\v', '\r', '\l', '\f'}
+  Whitespace* = {' ', '\t', '\v', '\r', '\n', '\f'}
     ## All the characters that count as whitespace (space, tab, vertical tab,
     ## carriage return, new line, form feed).
 
   Letters* = {'A'..'Z', 'a'..'z'}
     ## The set of letters.
 
+  Vowels* = {'a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'}
+    ## The set of vowels.
+  
+  Consonants* = Letters - Vowels
```

Comment:
update changelog.md

Reply:
Same as https://github.com/nim-lang/Nim/pull/18193/files#r646103498 ; `r`,`y`,`w`,`h`

Reply:
not sure what you mean by rywh (granted, you could say same about xyz, but i was referring to fact that symbol additions should be added to changelog)

and yes, i agree that Vowels and Consonents can't be added because of y and w

Reply:
Yep, removing them.